src/watchdog/test/data/no_condtion_pod.json

```diff
@@ -18,7 +18,7 @@
                 },
                 "ownerReferences": [
                     {
-                        "apiVersion": "extensions/v1beta1",
```

Comment:
It seems the spec.selector.matchLabels is missing in this ds.

Reply:
it's `ownerReferences`, not daemon set spec